plumbing/format/packfile/parser.go

```diff
@@ -256,8 +279,14 @@ func (p *Parser) indexObjects() error {
 			copy(p.deltas[oh.Offset], data)
 		}
 
+		if delta {
+			p.deltasTotal++
+		}
+
 		p.oiByOffset[oh.Offset] = ota
 		p.oi[i] = ota
+		p.objectsSeen++
+		p.writeObjectProgress()
```

Comment:
This has proven to be an effective place to count objects we're receiving, but I'm realizing that this may be more of a quick enumeration than actually representative of receiving the objects. Might give this a bit more thought.